modules/dcache/src/main/java/diskCacheV111/services/PoolStatisticsV0.java

```diff
@@ -338,26 +270,33 @@ public void getInfo(PrintWriter pw) {
         pw.println("   Stat Base : "+_dbBase);
         pw.println("   Html Base : "+_htmlBase);
         pw.println("      Images : "+ _images);
-        pw.println("    Next Run : "+_hourly);
+        pw.println("    Next Run : "+ Instant.now().plusSeconds(scheduledTask.getDelay(TimeUnit.SECONDS)));
```

Comment:
As `scheduledTask` get assigned in `started` method,  calling `getInfo` before scheduledTask is initialized will result in NullPointerException. Thus you have to check that `scheduledTask` is not null. Moreover, the `scheduledTask` should be `volatile` as it initialiaed and used from different threads.

Reply:
thanks for your feedback :) Do you think this simple if statement is already enough?

Reply:
yes, or something like:

```
 pw.println("    Next Run : " + (scheduledTask == null ? "-" : ZonedDateTime.now()...toString());
```

Reply:
[non-blocking]

Please bear in mind that the output here is for admins (i.e., human consumption).  An ISO 8601 representation is certainly valid, but not necessarily easy to digest.  You could use `org.dcache.util.TimeUtils#relativeTimestamp` to provide something easier for the admin to understand.

Reply:
pw.println("    Next Run : " +
                (scheduledTask == null ? "-" :
                TimeUtils.relativeTimestamp(Instant.now().plusSeconds(scheduledTask.getDelay(TimeUnit.SECONDS)))));

Thanks for your feedback, too, Paul. I have now taken both of your comments into account. The output should now be more readable. However the line of code is very long... Should I commit again or change something else?